mcc/trek/trek/trek_test.go

```diff
@@ -64,4 +65,26 @@ func TestAdd(t *testing.T) {
 	if err == nil {
 		t.Errorf("Both original and final should be present")
 	}
+	vhosts := "server {\n" +
```

Comment:
@mbernabe Maybe we could have a helper to build this, or using a template... Again, do you think this is valuable effort, now? I think not, while we don't need this anywhere else.